contrib/golang/common/go/api/filter.go

```diff
@@ -255,6 +261,12 @@ func (*EmptyUpstreamFilter) OnData(buffer []byte, endOfStream bool) FilterStatus
 func (*EmptyUpstreamFilter) OnEvent(event ConnectionEvent) {
 }
 
+type TcpUpstreamCallbackHandler interface {
+	FilterCallbackHandler
```

Comment:
Maybe we can introduce a new simple callback interface, since FilterCallbackHandler seems too much here.

Reply:
done, have replace it with a simpler callback.